app/controllers/api/items_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::ItemsController < ApiController
+  before_action :authenticated?
+  
+  def index
+    items = Item.all
+    render json: lists, each_serializer: ListSerializer
+  end
+  
+  def create
+    item = Item.new(item_params)
```

Comment:
if items belong to lists, you'll want to get the list id from the params hash and associate this item to that list